packages/plugin-conversation/src/share-activity.js

```diff
@@ -78,6 +78,20 @@ const ShareActivity = SparkPlugin.extend({
    * @returns {EventEmittingPromise}
    */
   add(file) {
+    let actions;
+
+    // ArrayBuffer might be located in an object with the actions.
+    if (has(file, `file`) && has(file, `actions`)) {
```

Comment:
ArrayBuffers do not have properties.  In order to store the actions within the file, this differentiation was necessary.  There is some discussion about that earlier in this PR discussion.

It is tough to efficiently communicate here, so lets talk on spark if you have further questions.  I would love some feedback.  Personally, I have only been able to come up with mediocre solutions to how to this challenge.